lib/rubocop/cop/rails/rake_environment.rb

```diff
@@ -41,8 +41,25 @@ def on_block(node)
           end
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            task_name = node.arguments[0]
+            task_dependency = corrected_task_dependency(task_name)
+
+            corrector.replace(task_name.loc.expression, task_dependency)
+          end
+        end
+
         private
 
+        def corrected_task_dependency(task_name)
```

Comment:
It would be better to be named `correct_task_dependency` as a small point.

```suggestion
        def correct_task_dependency(task_name)
```

Reply:
Done